server/src/test/java/org/elasticsearch/common/bytes/CompositeBytesReferenceTests.java

```diff
@@ -167,12 +167,40 @@ public void testGetIntLE() {
     }
 
     public void testGetDoubleLE() {
-        // first bytes array = 1.2, second bytes array = 1.4, third bytes array = 1.6
-        BytesReference[] refs = new BytesReference[] {
-            new BytesArray(new byte[] { 0x33, 0x33, 0x33, 0x33, 0x33, 0x33, -0xD, 0x3F }),
-            new BytesArray(new byte[] { 0x66, 0x66, 0x66, 0x66, 0x66, 0x66, -0xA, 0x3F }),
-            new BytesArray(new byte[] { -0x66, -0x67, -0x67, -0x67, -0x67, -0x67, -0x7, 0x3F }) };
-        BytesReference comp = CompositeBytesReference.of(refs);
+        // first double = 1.2, second double = 1.4, third double = 1.6
+        byte[] data = new byte[] {
+            0x33,
+            0x33,
+            0x33,
+            0x33,
+            0x33,
+            0x33,
+            -0xD,
+            0x3F,
+            0x66,
+            0x66,
+            0x66,
+            0x66,
+            0x66,
+            0x66,
+            -0xA,
+            0x3F,
+            -0x66,
+            -0x67,
+            -0x67,
+            -0x67,
+            -0x67,
+            -0x67,
+            -0x7,
+            0x3F };
```

Comment:
I wonder if you could do:
```
byte[] data = new byte[3 * Double.BYTES];
ByteUtils.writeDoubleLE(data, 1.2, 0);
ByteUtils.writeDoubleLE(data, 1.4, Double.BYTES);
ByteUtils.writeDoubleLE(data, 1.6, 2 * Double.BYTES);
```

Would that be more readable? I'm not really sure.